spec/support/seeds/seeds_e2e.rb

```diff
@@ -563,3 +563,42 @@
     category: :bug,
   )
 end
+
+##############################################################################
+
+seeder.create_if_doesnt_exist(Podcast, "title", "Test podcast") do
```

Comment:
Creating a seeded podcast for E2E testing was discussed in https://github.com/forem/forem/pull/14139 - this is purely a copy/paste of what has been added in that PR 🙂